src/labkey/Domain.ts

```diff
@@ -442,23 +479,75 @@ export function listDomains(config: ListDomainsOptions): XMLHttpRequest {
 }
 
 interface GetPropertiesParams {
+    /**
+     * Get properties from the given set of domains.
+     */
+    domainIds?: number[]
+    /**
+     * Get properties from the given set of domain kinds. For example, "Vocabulary" or "SampleSet"
+     */
+    domainKinds?: string[]
+    /**
+     * Get properties matching the query filters.
+     * For example, `query.measure~eq=true` will find all measure properties.
+     */
+    filters?: string[]
```

Comment:
I think we should try to use `Filter.IFilter`s here instead so it's not left up to the user to compose the filter as a string.